iris_core/modules/core/entity/entity_views.js

```diff
@@ -136,43 +136,21 @@ iris.modules.entity.registerHook("hook_frontend_template_parse", 0, function (th
  */
 iris.modules.entity.registerHook("hook_entity_created", 0, function (thisHook, entity) {
 
-  iris.hook("hook_entity_view", thisHook.authPass, null, entity).then(function (filtered) {
-
-    iris.hook("hook_entity_view_" + entity.entityType, thisHook.authPass, null, filtered).then(function (filtered) {
-
-      send(filtered);
-
-    }, function (fail) {
-
-      if (fail === "No such hook exists") {
-
-        send(filtered);
+    for (var authUser in iris.modules.auth.globals.userList) {
```

Comment:
Oops. Not sure about that. Started reading through other PRs and there are a mixed set of styles. Both double and single spacing. We need a style guide.